src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/profiler/model/ProfileOperationResponse.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * ProfileOperationResponse.java
+ *
+ * Copyright (C) 2009-16 by RStudio, Inc.
+ *
+ * Unless you have received this program directly from RStudio pursuant
+ * to the terms of a commercial license agreement with RStudio, then
+ * this program is licensed to you under the terms of version 3 of the
+ * GNU Affero General Public License. This program is distributed WITHOUT
+ * ANY EXPRESS OR IMPLIED WARRANTY, INCLUDING THOSE OF NON-INFRINGEMENT,
+ * MERCHANTABILITY OR FITNESS FOR A PARTICULAR PURPOSE. Please refer to the
+ * AGPL (http://www.gnu.org/licenses/agpl-3.0.txt) for more details.
+ *
+ */
+
+package org.rstudio.studio.client.workbench.views.source.editors.profiler.model;
+
+import com.google.gwt.core.client.JavaScriptObject;
+
+public class ProfileOperationResponse extends JavaScriptObject
+{
+   protected ProfileOperationResponse()
+   {
+   }
+   
+   public final native String getErrorMessage() /*-{
+      return this.error ? this.error.message.join(' ') : null;
```

Comment:
Is this intended to work around the fact that the R -> JSON serializer wraps objects in arrays by default? If so, it's usually simpler to wrap the object in `.rs.scalar` to signal to the serializer instead (e.g. `list(error = .rs.scalar(err))`)